dockerfile-configs/common-components.yaml

```diff
@@ -117,6 +117,13 @@
     command: --chown=65532:65532
     info: Bash script which installs the `etcdctl` tool in the container. If you are running this container as non-root, execute the `install-etcdctl` script in the `/nonroot/hacks` directory.
 
+- copy:
+  - name: print-etcd-cheatsheet
+    from: ./hacks/print-etcd-cheatsheet.sh
+    to: /nonroot/hacks/print-etcd-cheatsheet
+    command: --chown=65532:65532
+    info: Bash script which prints out etcd certificate paths and handy etcdctl comands. If you are running this container as a non-root container, execute the script in the `/nonroot/hacks` directory. Else use the one in `/hacks`
```

Comment:
in my case I got the following error when running the script:
error: cannot find the process id for etcd-wrapper. Please ensure it is running

how to ensure that it is running / what should I do? :)

Reply:
This needs a running etcd process
easiest way is to use one of our dev clusters and mount this image as an ephemeral image onto the etcd pod 
(eg: `k debug -it etcd-main-0 --image=< ops-toolbelt image >`)

Reply:
I think it would be helpful to add this to the info, except this is obvious for everyone except me :D

Reply:
Updated info to mention this 
Hope this is okay